spec/factories/teams.rb

```diff
@@ -1,6 +1,6 @@
 FactoryGirl.define do
   factory :team do
     sequence(:name) { |n| "Patriots#{n}" }
-    city "New England"
+    location "New England"
```

Comment:
Cool! Yeah I think using some kind of continuous integration will be helpful to use. As an added perk, I'll then have some experience with CI!